From 018574553a0ad8b80b9eeb9e224810d34a88cb51 Mon Sep 17 00:00:00 2001
From: sub77 <sub77@ymail.com>
Date: Thu, 15 Dec 2016 01:48:04 +0000
Subject: [PATCH 25/29] Revert "N-Extras: Add dynamic theme BootAnimation
 support"

This reverts commit a7fd12daf8d1221803eb8c15ab479beace1dfeb6.

Signed-off-by: sub77 <sub77@ymail.com>
---
 cmds/bootanimation/BootAnimation.cpp | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/cmds/bootanimation/BootAnimation.cpp b/cmds/bootanimation/BootAnimation.cpp
index e0359f8..c6b2f63 100644
--- a/cmds/bootanimation/BootAnimation.cpp
+++ b/cmds/bootanimation/BootAnimation.cpp
@@ -68,7 +68,6 @@ namespace android {
 static const char OEM_BOOTANIMATION_FILE[] = "/oem/media/bootanimation.zip";
 static const char SYSTEM_BOOTANIMATION_FILE[] = "/system/media/bootanimation.zip";
 static const char SYSTEM_ENCRYPTED_BOOTANIMATION_FILE[] = "/system/media/bootanimation-encrypted.zip";
-static const char THEME_BOOTANIMATION_FILE[] = "/data/system/theme/bootanimation.zip";
 static const char SYSTEM_DATA_DIR_PATH[] = "/data/system";
 static const char SYSTEM_TIME_DIR_NAME[] = "time";
 static const char SYSTEM_TIME_DIR_PATH[] = "/data/system/time";
@@ -414,11 +413,6 @@ status_t BootAnimation::readyToRun() {
     }
     else if (access(getAnimationFileName(IMG_SYS), R_OK) == 0) {
         mZipFileName = getAnimationFileName(IMG_SYS);
-    else if (access(THEME_BOOTANIMATION_FILE, R_OK) == 0) {
-        mZipFileName = THEME_BOOTANIMATION_FILE;
-    }
-    else if (access(OEM_BOOTANIMATION_FILE, R_OK) == 0) {
-        mZipFileName = OEM_BOOTANIMATION_FILE;
     }
 
 #ifdef PRELOAD_BOOTANIMATION
-- 
2.1.4

